contracts/upgradeable_contracts/U_HomeBridge.sol

```diff
@@ -16,18 +16,24 @@ contract HomeBridge is EternalStorage, Validatable {
         address _validatorContract,
         uint256 _homeDailyLimit,
         uint256 _maxPerTx,
-        uint256 _minPerTx
+        uint256 _minPerTx,
+        uint256 _foreignGasPrice,
```

Comment:
this should not be called `_foreignGasPrice` since it actually reflects gas price which should be used for transactions sent to Home Bridge.

Reply:
ok